### PR TITLE
IMED-01: created Drug class and list item layout

### PR DESCRIPTION
**Pull request by Saotomefirst**
**Changes:**

- Added Drug class
- Created mock list on Main Activity with some records for further testing
- Added Drug Item Card

image

### CODE REVIEW

File: app/src/main/java/com/saotome/imed_app/MainActivity.kt [hunk 1]
```diff
@@ -32,6 +35,15 @@ class MainActivity : AppCompatActivity() {
             Snackbar.make(view, "Replace with your own action", Snackbar.LENGTH_LONG)
                 .setAction("Action", null).show()
         }
+
+        loadDrugMockList()
+
```
Comment (maylcf) [inline app/src/main/java/com/saotome/imed_app/MainActivity.kt]: Please remove empty line

File: app/src/main/res/layout/item_drug.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    app:cardCornerRadius="10dp"
+    app:cardElevation="10dp"
+    android:layout_margin="8dp">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_margin="16dp"
+        android:elevation="10dp">
+
```
Comment (maylcf) [inline app/src/main/res/layout/item_drug.xml]: Please remove empty line.

File: app/src/main/res/layout/item_drug.xml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        <TextView
+            android:id="@+id/tv_start_time"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textColor="@color/black"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/tv_dosage"
+            tools:text="Starting 10:00" />
+
```
Comment (maylcf) [inline app/src/main/res/layout/item_drug.xml:55]: 2 lines here.

File: app/src/main/res/layout/item_drug.xml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    app:cardCornerRadius="10dp"
+    app:cardElevation="10dp"
+    android:layout_margin="8dp">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_margin="16dp"
```
Comment (maylcf) [inline app/src/main/res/layout/item_drug.xml]: Please use dimen values in `dimens.xml`. If the value you are looking for is not there yet, please add.

Comment (Saotomefirst) [reply app/src/main/res/layout/item_drug.xml]: Done as per requested.
141d5a64a90714933f27376439b988545347c2e9

**Review by maylcf — CHANGES_REQUESTED** [review]
Small changes

File: app/src/main/java/com/saotome/imed_app/MainActivity.kt (hunk 1 above)
Comment (Saotomefirst) [reply app/src/main/java/com/saotome/imed_app/MainActivity.kt]: done.
7f7bfaf442af1833a0483254e051e577d7c89515

File: app/src/main/res/layout/item_drug.xml (hunk 2 above)
Comment (Saotomefirst) [reply app/src/main/res/layout/item_drug.xml]: Done.
7f7bfaf442af1833a0483254e051e577d7c89515

File: app/src/main/res/layout/item_drug.xml (hunk 3 above)
Comment (Saotomefirst) [reply app/src/main/res/layout/item_drug.xml:55]: Done.
7f7bfaf442af1833a0483254e051e577d7c89515

**Review by maylcf — APPROVED** [review]
👍

**Review by maylcf — CHANGES_REQUESTED** [review]
test

**Review by maylcf — APPROVED** [review]

**Pull request merged**